src/validation/fields/date.validation.ts

```diff
@@ -17,7 +17,8 @@ import {
   MonthFieldErrors,
   YearFieldErrors,
   checkDateIPLegalEntityBO,
-  checkCeasedDateOnOrAfterStartDate
```

Comment:
we could just define checkCeasedDateOnOrAfterStartDate in terms of checkFirstDateOnOrAfterSecondDate and leave callers as is. Might be clearer too.